src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -91,6 +88,17 @@ function MetadataSidebarRedesign({
         setEditingTemplate(selectedTemplate);
     };
 
+    const checkIfTemplateInstanceExists = (id: string): MetadataTemplateInstance => {
+        return templateInstances.find(templateInstance => templateInstance.id === id);
```

Comment:
So it got me thinking... in `createTemplateInstance` we are setting `id: template.id`, so each `MetadataTemplateInstance` has `template.id` as it's `id`. We I guess we should use `instance.id` instead - can you update definition of `createTemplateInstance` please? We could have:
```
id: instance.id
templateId: template.id
```
instead of 
```
id: template.id
```

WDYT? @wpiesiak @karolinaru

Reply:
I also wonder when `instance.id` is created if user creates new instance from template?

Reply:
Anyway you want to compare `template.id` here if I understand correctly :) But above mentioned change is needed anyway :)